.github/workflows/update_aas_extension.yml

```diff
@@ -0,0 +1,92 @@
+name: Update extension on AAS, triggered by dd-trace-dotnet nightly build
+
+on:
+  workflow_dispatch:
+    inputs:
+      sha:
+        description: 'Hash commit of dd-trace-dotnet build'
+        required: true
+  repository_dispatch:
+    types: [dd-trace-dotnet-release]
```

Comment:
Change this one to dd-trace-dotnet-nightly for instance

Reply:
renamed in 852851287bb3961484816f0401baaf97bdab7546